releasenotes/notes/agent-port-conflict-diagnose-suite-1b35404ce0f60a2b.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Introducing the port conflict suite to the Agent diagnose command. 
+    Adding this for MacOS only first. Will incrementally add support for Linux and Windows. 
+    The Agent will need access to read from the host's ports.
```

I don't think the agent needs permission to read from the ports, it's rather info about the processes
```suggestion
    The Agent might need particular permissions to read information about processes using ports.
```